cassandra/pom.xml

```diff
@@ -27,13 +27,14 @@
     </parent>
 
     <groupId>org.apache.zeppelin</groupId>
-    <artifactId>zeppelin-cassandra_2.10</artifactId>
+    <artifactId>zeppelin-cassandra_${scala.binary.version}</artifactId>
```

Comment:
I don't have preference, but doesn't [dev/change_scala_version.sh](https://github.com/apache/zeppelin/blob/master/dev/change_scala_version.sh) take care of artifact name?